core/src/com/unciv/ui/worldscreen/WorldScreenTopBar.kt

```diff
@@ -143,9 +145,12 @@ class WorldScreenTopBar(val worldScreen: WorldScreen) : Table() {
         return menuButton
     }
 
-    private fun getOverviewButton(): TextButton {
-        val overviewButton = "Overview".toTextButton()
-        overviewButton.labelCell.pad(10f)
+    private fun getOverviewButton(): Button {
+        val overviewButton = Button(CameraStageBaseScreen.skin)
+        overviewButton.add("Overview".toLabel()).pad(10f)
+        if (Gdx.app.input.isPeripheralAvailable(Input.Peripheral.HardwareKeyboard)) {
+            overviewButton.add(Label("(E)", CameraStageBaseScreen.skin).apply { color = Color.LIGHT_GRAY }).padLeft(0f)
```

Comment:
("E").toLabel(color = Color.LIGHT_GRAY)

Reply:
Didn't want it to go through the translation engine... But needs comparison with unit commands code anyway (different color at least over there), and possibly meaningless depending on a consensus about my "similar hints for other worldscreen buttons would look horrible" comment. Honestly, I'd _not_ change the button for this.